src/monorepo-shipit/bin/shipit.js

```diff
@@ -15,19 +16,41 @@ import createPushPhase from '../src/phases/createPushPhase';
 import type { Phase } from '../types.flow';
 
 // yarn monorepo-babel-node src/monorepo-shipit/bin/shipit.js
-// yarn monorepo-babel-node src/monorepo-shipit/bin/shipit.js --glob-pattern="/abacus.js"
-
-const options = commandLineArgs([
-  {
-    // This option allows us to grep subset of config files. Especially useful when running the
-    // Shipit binary locally for testing purposes.
-    name: 'glob-pattern',
-    type: String,
-    defaultValue: '/*.js',
-  },
-]);
-
-iterateConfigs(options['glob-pattern'], (config) => {
+// yarn monorepo-babel-node src/monorepo-shipit/bin/shipit.js --config-filter="/abacus.js"
+
+const options = commandLineArgs(
+  [
+    {
+      // This option allows us to grep subset of config files. Especially useful when running the
+      // Shipit binary locally for testing purposes.
+      name: 'config-filter',
+      type: String,
+      defaultValue: '/*.js',
+    },
+    {
+      name: 'config-dir',
+      type: String,
+      defaultValue: './.shipit',
```

Comment:
I think this is a great idea. Could you please update a README.md to recommend this approach? I think we should update README in general to explain the new `--committer-name` and `--committer-email` options (usage in general).

Reply:
Done. To swing back on committer-name and committer email - is this in essence what will commit to the destination repo, but not the author? 

So take your Flow example you linked

<img width="553" alt="image" src="https://github.com/adeira/universe/assets/6801309/9afb0839-87b6-4c8a-befa-27fd5f28b4fc">

Us setting committer-name/email is in essence setting facebook-github-bot, yeah?

Reply:
That is correct. The author is preserved, and the committer config is for `facebook-github-bot`. Here is another example where the committer is `adeira-github-bot`: https://github.com/adeira/sx/commit/10760b1ee4830b0a2e028c501157163ca68b83c4